tpp-prophets/interprophet.xml

```diff
@@ -1,59 +1,31 @@
     </requirements>
-    <command>
+    <command detect_errors="aggressive">
 <![CDATA[
         export XML_ONLY="true";
 
         interprophet.rb --galaxy
-        #if str( $input_type.type ) == "files":
-            #for $pepxml_file in $input_type.pepxml_files:
-                '${pepxml_file}'
-            #end for
-        #else
-            #for $list_item in $input_type.pepxml_list:
-                '${list_item}'
-            #end for
-            #for $extra_list_holder in $input_type.extra_pepxml_lists:
-                #for $list_item in $extra_list_holder.extra_list:
-                    '${list_item}'
-                #end for
-            #end for
-        #end if
-
-            -o interprophet_output.pep.xml
-            $use_nss
-            $use_nrs
-            $use_nse
-            $use_nsi
-            $use_nsm
-            --p-thresh $p_thresh
-            --threads "\${GALAXY_SLOTS:-12}"
+        #for $input_file in $input_files:
+            '${input_file}'
+        #end for
+        -o interprophet_output.pep.xml
+        $use_nss
+        $use_nrs
+        $use_nse
+        $use_nsi
+        $use_nsm
+        --p-thresh $p_thresh
+        --threads "\${GALAXY_SLOTS:-12}"
 
 ]]>
     </command>
     <inputs>
 
-        <conditional name="input_type">
-            <param name="type" type="select" label="Input Type">
-                <option value="files" selected="true">Datasets</option>
-                <option value="collections">Collections</option>
-            </param>
-            <when value="files">
-                <param name="pepxml_files" multiple="true" type="data" format="peptideprophet_pepxml" label="Peptide Prophet Results" help="These files will typically be outputs from search tools that have subsequently been run through peptide prophet"/>
-            </when>
-            <when value="collections">
-                <param name="pepxml_list" type="data_collection" multiple="true" collection_type="list" format="peptideprophet_pepxml" label="Peptide Prophet Results" help="These files will typically be outputs from search tools that have subsequently been run through peptide prophet"/>
-                <repeat name="extra_pepxml_lists" title="Additional PepXML Collections" >
-                    <param name="extra_list" type="data_collection" multiple="true" collection_type="list" format="peptideprophet_pepxml" label="Peptide Prophet Results" help="These files will typically be outputs from search tools that have subsequently been run through peptide prophet"/>
-                </repeat>
-            </when>
-        </conditional>
-
-        <!-- <param name="pepxml_files" multiple="true" type="data" format="peptideprophet_pepxml" label="Peptide Prophet Results" help="These files will typically be outputs from search tools that have subsequently been run through peptide prophet"/> -->
+        <param name="input_files" multiple="true" type="data" format="peptideprophet_pepxml"
```

Comment:
Ditto comment above. Have you tested with a multifile workflow?  I'm going to trust that you know this much better than I